tests/idcindex.py

```diff
@@ -37,68 +38,107 @@ def test_get_idc_version(self):
+                if output_format == "list":
+                    self.assertIsInstance(patients, list)
+                    self.assertTrue(bool(patients))  # Check that the list is not empty
+                elif output_format == "dict":
+                    self.assertTrue(
+                        isinstance(patients, dict)
+                        or (
+                            isinstance(patients, list)
+                            and all(isinstance(i, dict) for i in patients)
+                        )
+                    )  # Check that the output is either a dictionary or a list of dictionaries
+                    self.assertTrue(
+                        bool(patients)
+                    )  # Check that the output is not empty
+                elif output_format == "df":
+                    self.assertIsInstance(patients, pd.DataFrame)
+                    self.assertFalse(
+                        patients.empty
+                    )  # Check that the DataFrame is not empty
 
     def test_get_studies(self):
-        studies = self.client.get_dicom_studies(
-            patientId="PCAMPMRI-00001", outputFormat="list"
-        )
-
-        self.assertIsNotNone(studies)
+        # Define the values for each optional parameter
+        output_format_values = ["list", "dict", "df"]
+        patient_id_values = ["PCAMPMRI-00001", ["PCAMPMRI-00001", "NoduleLayout_1"]]
 
-        studies = self.client.get_dicom_studies(
-            patientId=["PCAMPMRI-00001", "NoduleLayout_1"], outputFormat="list"
-        )
+        # Test each combination
+        for patient_id in patient_id_values:
+            for output_format in output_format_values:
+                studies = self.client.get_dicom_studies(
+                    patientId=patient_id, outputFormat=output_format
+                )
 
-        self.assertIsNotNone(studies)
+                # Check if the output format matches the expected type
+                if output_format == "list":
+                    self.assertIsInstance(studies, list)
+                    self.assertTrue(bool(studies))  # Check that the list is not empty
+                elif output_format == "dict":
+                    self.assertTrue(
+                        isinstance(studies, dict)
+                        or (
+                            isinstance(studies, list)
+                            and all(isinstance(i, dict) for i in studies)
+                        )
+                    )  # Check that the output is either a dictionary or a list of dictionaries
+                    self.assertTrue(bool(studies))  # Check that the output is not empty
+                elif output_format == "df":
+                    self.assertIsInstance(studies, pd.DataFrame)
+                    self.assertFalse(
+                        studies.empty
+                    )  # Check that the DataFrame is not empty
 
     def test_get_series(self):
-        """
```

Comment:
I intentionally left this comment so that it can be reused later if needed, and also help with transparency of selecting this specific series. Is there a good reason to remove this comment?

Reply:
I did not mean to delete it. I'll place it back.